### PR TITLE
Update netcore3 engineering plans for aspnet

### PR DESCRIPTION
**Pull request by natemcmaster**
Updating the plan to indicate that there are actually 3 aspnet repos to track in this plan.

### CODE REVIEW

File: Documentation/NetCore3EngineeringRepoStatus.md [hunk 1]
```diff
@@ -188,6 +192,28 @@ Owners: licavalc
 | [Engineering dependency flow](https://github.com/dotnet/arcade/issues/673)       | ?                    | Scheduled   | Planning to start mid-November |
 | [Internal builds from dnceng](https://github.com/dotnet/arcade/issues/675)       | ?                    | In progress | |
 
+#### EntityFrameworkCore
+
+<https://github.com/aspnet/EntityFrameworkCore>
+
+| Work stream                                                                      | Est. Completion Date | Status      | Notes |
+| -------------------------------------------------------------------------------- |:--------------------:| ----------- | ----- |
+| [Using Azure DevOps for CI](](???))         | ?                    | Scheduled   | Planning to start mid-November |
```
Comment (natemcmaster) [inline Documentation/NetCore3EngineeringRepoStatus.md]: @chcosta do you have a script/tool to generate the issues for each repo?

File: Documentation/NetCore3EngineeringRepoStatus.md [hunk 2]
```diff
@@ -14,14 +14,16 @@ Target completion date for these workstreams is 12/14/2018.
 | Repo                                          | Owner                         | Status   | Risk Assessment                                                                           | Completion Status | Plan | Notes |
 | --------------------------------------------- | ----------------------------- | -------- |:-----------------------------------------------------------------------------------------:|:-----------------:| ----- |----- |
 | [Arcade](#arcade)                             | [mawilkie](#mark-wilkie)      | Complete | ![done](https://findicons.com/files/icons/767/wp_woothemes_ultimate/16/checkmark.png)     | 4 / 4             | | |
-| [ASP.Net](#aspnet)                            | [namc](#nate-mcmaster)        | At risk  | ![negative](http://individual.icons-land.com/IconsPreview/Sport/PNG/16x16/Ball_Red.png)   | 0 / 4             | | Internal builds in dnceng work is in progress - Need plan |
+| [AspNetCore](#aspnet)                         | [namc](#nate-mcmaster)        | At risk  | ![negative](http://individual.icons-land.com/IconsPreview/Sport/PNG/16x16/Ball_Red.png)   | 0 / 4             | | Internal builds in dnceng work is in progress - Need plan |
```
Comment (Eilon) [inline Documentation/NetCore3EngineeringRepoStatus.md]: For the sake of maintaining my own sanity, can we fake-call these entries `aspnet-AspNetCore`, `aspnet-EntityFrameworkCore`, etc.? I realize that's fake, but it's not like it matters from a technical perspective.

File: Documentation/NetCore3EngineeringRepoStatus.md (hunk 1 above)
Comment (chcosta) [reply Documentation/NetCore3EngineeringRepoStatus.md]: Yeah, I wrote a simple tool that took 5 issues I originally created and used them as a template to create issues for every repo I cared about.

Comment (natemcmaster) [reply Documentation/NetCore3EngineeringRepoStatus.md]: Ok. Can you use it to create issues for aspnet/Extensions and aspnet/EntityFramworkCore too? I can update the PR with links to those items.

File: Documentation/NetCore3EngineeringRepoStatus.md (hunk 2 above)
Comment (natemcmaster) [reply Documentation/NetCore3EngineeringRepoStatus.md]: fine with me.

File: Documentation/NetCore3EngineeringRepoStatus.md (hunk 1 above)
Comment (chcosta) [reply Documentation/NetCore3EngineeringRepoStatus.md]: Extensions - https://github.com/dotnet/arcade/issues?utf8=%E2%9C%93&q=is:open+label:Engineering-ecosystem+extensions
EntityFrameworkCore - https://github.com/dotnet/arcade/issues?utf8=%E2%9C%93&q=is:issue+is:open+label:Engineering-ecosystem+EntityFrameworkCore

Comment (natemcmaster) [reply Documentation/NetCore3EngineeringRepoStatus.md]: Thanks.

**Pull request merged**